### PR TITLE
Optimize second paragraph of Chinese version

### PR DESCRIPTION
**Pull request by wahyd4**
Try to make the last sentence of the second paragraph easier to understand.

### CODE REVIEW

**Comment by 10000TB** [comment]
LGTM.

@wahyd4  Thank you!

File: code-of-conduct-ZH-Simple.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 我们致力于为每个人提供一个没有扰乱，包容的体验，我们绝不容忍参与者对社区得任何形式的扰乱。我们要求参与者相互体谅，举止专业，并且对他人充满尊重。这套行为准则及其相关条例同样适用于开源社区活动范围以外的令人无法接受的行为，这包括所有的社区场所：在线，面对面，以及任何其他形式的点对点的交流，以及任何地点以至于相关的行为可以对社区成员的安全和个人健康造成伤害。Docker, Inc（Docker峰会，线下见面，用户群组 ）组织的或在Docker, Inc 场所所举办的活动的参展者，发言者，赞助者,工作人员，以及所有其他的参与者同样受到社区指南和行为准则的制约。
 
-多元化和包容精神使得Docker社区更加强大和繁荣。我们鼓励来自尽可能不同的，多元的背景的人士的参与, 并且我们希望我们的立场对于每个人都非常鲜明。
+多元化和包容精神使得Docker社区更加强大和繁荣。我们鼓励来自尽可能不同的，多元的背景的人士的参与, 并且希望所有人都非常清楚我们的鲜明立场。
```
Comment (AntaresS) [inline code-of-conduct-ZH-Simple.md]: `我们鼓励来自尽可能不同的，多元的背景的人士的参与`过于冗余，建议改为 `我们鼓励来自多元化背景人士的参与`，多元在词义上包含了不同。

**Comment by AntaresS** [comment]
Overall looks good. Just some minor tweaks.

File: code-of-conduct-ZH-Simple.md (hunk 1 above)
Comment (wahyd4) [reply code-of-conduct-ZH-Simple.md]: Looks good to me

**Pull request merged**